demo/index.html

```diff
@@ -1380,21 +1370,30 @@
               })
             }
 
-            hx.theme.plot.colors.forEach(function(col, i) {
-              axis.addSeries('line', {
-                title: 'Series ' + (i+1),
-                data: createData(randomSign(), randomSign(), randomSign()),
-                labelInterpolated: true,
-                markersEnabled: true,
-                strokeEnabled: true,
-                strokeColor: col,
-                fillEnabled: true,
-                fillColor: hx.color(col).alpha(0.2).toString(),
-                group: 'some-group'
-              })
-            })
+            const axes = [{
```

Comment:
Does using const here have any implications? The rest of the page was not written with ES6 in mind so might be worth using `var` here